sootup.callgraph/src/test/java/sootup/callgraph/TestExample2.java

```diff
@@ -0,0 +1,32 @@
+package sootup.callgraph;
+
+class UpdatedThreadInner extends Thread {
+  @Override
+  public void run() {
+    System.out.println("TestExample2 run() executed.");
+  }
+  //    @Override
+  //    public synchronized void start() {
+  //        super.start();
+  //        System.out.println("TestExample2 start() executed.");
+  //    }
+}
+
+class UpdatedThreadOuter extends UpdatedThreadInner {
+  @Override
+  public synchronized void start() {
+    super.start();
+    System.out.println("TestExample2 start() executed.");
+  }
+  //    @Override
+  //    public void run() {
+  //        System.out.println("TestExample2 run() executed.");
+  //    }
+}
+
+class TestExample2 {
+  public static void main(String[] args) {
+    UpdatedThreadInner thread = new UpdatedThreadInner();
+    thread.start();
+  }
+}
```

Comment:
move this classes to sootup.callgraph/src/test/resources/callgraph/Implicit/source/